tests/test_flip.py

```diff
@@ -31,12 +32,17 @@ def test_invalid_inputs(self, _, spatial_axis, raises):
 
     @parameterized.expand(VALID_CASES)
     def test_correct_results(self, _, spatial_axis):
-        flip = Flip(spatial_axis=spatial_axis)
-        expected = []
-        for channel in self.imt[0]:
-            expected.append(np.flip(channel, spatial_axis))
-        expected = np.stack(expected)
-        self.assertTrue(np.allclose(expected, flip(self.imt[0])))
+        for p in TEST_NDARRAYS:
+            im = p(self.imt[0])
+            flip = Flip(spatial_axis=spatial_axis)
+            expected = []
+            for channel in self.imt[0]:
+                expected.append(np.flip(channel, spatial_axis))
+            expected = np.stack(expected)
+            result = flip(im)
+            if isinstance(result, torch.Tensor):
+                result = result.cpu()
+            self.assertTrue(np.allclose(expected, result))
```

Comment:
could you help change this to `np.testing.assert_allclose`([doc](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html)) it would give us more info when there are mismatches. also, as we are having more transform supporting both tensor and np.array, would be great to have a utility `assert_allclose` that can handle both types